app/assets/javascripts/miq_tree.js

```diff
@@ -561,6 +562,7 @@ window.miqInitTree = function(options, tree) {
     onNodeCollapsed: function(event, node) {
       miqTreeState(options.tree_name, node.key, false);
     },
+    // TODO
```

Comment:
TODO what? :)

Reply:
![](https://i.redd.it/y14xp8glyab31.jpg)